cmd/svcat/binding/bind_cmd.go

```diff
@@ -68,6 +69,9 @@ func NewBindCmd(cxt *command.Context) *cobra.Command {
 		"",
 		"The name of the binding. Defaults to the name of the instance.",
 	)
+	cmd.Flags().StringVar(&bindCmd.externalID, "external-id", "",
+		"The external ID for the binding for use with OSB API (Optional)",
```

Comment:
This text is invalid, external id is not used with OSBAPI.

Reply:
The ExternalID of ServiceBInding is sent as BindingID in OSB BindRequest.
https://github.com/kubernetes-incubator/service-catalog/blob/2e1ca7f8cf2b5d45b0542602f0d12cb18bb05f98/pkg/controller/controller_binding.go#L1038

Also, the comment for the ExternalID field of ServiceBinding says the same thing.
https://github.com/kubernetes-incubator/service-catalog/blob/2e1ca7f8cf2b5d45b0542602f0d12cb18bb05f98/pkg/apis/servicecatalog/types.go#L962-L965